brianlong/www.validators.app
# 181570523_adjust_api_to_data_centers

#### What's this PR do?
- Adjust API to use data_centers instead of ips

#### How should this be manually tested?
- Check API responses (data and performance; best to do that locally is to switch between branch master and this one)
- Best QA is possible on staging since it has all the data needed (all data_centers)

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181570523

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
